src/config.rs

```diff
@@ -27,6 +28,7 @@ impl<T> SubscriberConfig<T> {
     ) -> SubscriberConfig<T> {
         SubscriberConfig {
             env: Environment::Dev,
+            country: None,
```

Comment:
What about setting this default value with the `COUNTRY` env var? I think that the `COUNTRY` env var is a standard for all the containers

Reply:
I don't know... on the one hand, I think it could make sense, but why not read `env` from env too?

Also, since `COUNTRY` may not have a value I still have to default to `None` in case something goes bad, but that's another problem

Reply:
I think I agree with simone on setting it from env

> why not read env from env too?

actually, why not? We currently default to dev, how about we default to using the environment variable and then fall back to dev if unset?

Reply:
I think I agree with simone on setting it from env

>why not read env from env too?

actually, why not? We currently default to dev, how about we default to using the environment variable and then fall back to dev if unset?

Reply:
Ok!